input.cpp

```diff
@@ -12,80 +12,95 @@
 
 // functions that read from input files
 namespace marxan {
+    void to_lower(std::string& str)
+    {
+        transform(str.begin(), str.end(), str.begin(), ::tolower);
+    }
 
-    string storeFieldName(const vector<string>& varlist, int numvars, string sVarName,
-        const vector<string>& head, string fname) {
-
-        // add a field name from an input file header to a list
-        int foundit = 0;
-        int temp, newlink;
-
-        for (int i = 0; (i < numvars && foundit == 0); i++)
-        {
-            if (varlist[i].compare(sVarName) == 0)
-                foundit++;
-        }
-
-        if (!head.empty() && find(head.begin(), head.end(), sVarName) != head.end())
+    vector<string> get_tokens(const std::string& str)

```

Comment:
Could you add some unit tests to this? Also I think we should move it to utils since it's more of a general function.

Reply:
Agree. Moving to utils postponed until utils.cpp added to the project.

Reply:
There is already a utils.hpp in the project where similar functions sit